test/utilities/context-store.js

```diff
@@ -0,0 +1,39 @@
+class ContextStore {
```

Comment:
This could also be a [jest manual mock](https://jestjs.io/docs/manual-mocks) which would benefit from built in mock helpers like call assertions.